aten/src/ATen/native/quantized/cpu/qnnpack_maxpool.cpp

```diff
@@ -0,0 +1,157 @@
+#include <ATen/ATen.h>
+#include <ATen/core/op_registration/op_registration.h>
+#include <aten/src/ATen/native/Pool.h>
+#include <caffe2/utils/threadpool/ThreadPoolMobile.h>
+
+#include "init_qnnpack.h"
+#include "qnnpack_utils.h"
+
+namespace at {
+namespace native {
+namespace {
+
+class QNNPACKMaxPool2D final : public torch::OperatorKernel {
+ public:
+#ifdef USE_QNNPACK
+  Tensor operator()(
+      Tensor input,
+      std::vector<int64_t> kernel_size,
```

Comment:
shall we use "torch::List<int64_t>" instead? seems there was an effort to unify it to torch::List but @smessmer might be expert to comment.

Reply:
Yes, please use torch::List.